dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/Entity.java

```diff
@@ -56,27 +56,27 @@
     private Integer index;
 
     /**
-     * UID of object (if object is id object).
+     * UID of entity (if object is id object).
      */
     @JsonProperty
     private String uid;
 
     private List<TrackerErrorReport> errorReports = new ArrayList<>();
 
-    public TrackerObjectReport( TrackerType trackerType )
+    public Entity( TrackerType trackerType )
     {
         this.trackerType = trackerType;
     }
 
-    public TrackerObjectReport( TrackerType trackerType, String uid, Integer index )
+    public Entity( TrackerType trackerType, String uid, Integer index )
     {
         this.trackerType = trackerType;
```

Comment:
Should we use the `TrackerDto` or at least accept it in the constructor? Maybe that makes things easier. The TrackerDto is the type and its UID.

Reply:
I think we will need to rethink the whole Entity concept in the Report. At the moment we are being really verbose just to give the client a list of UIDs by type. That is actually a list of TrackerDtos